documentation/contributing/contributor-guide.adoc

```diff
@@ -10,6 +10,25 @@ Before you begin you need to:
 
 . xref:setting-up-git[Install and set up Git]
 
+[[contribution-accessibility]]
+== Accessibility
+
+Strimzi aims to make its web content accessible.
+When contributing to the documentation, ensure that your content adheres to the following:
+
+* Images:
+** Include a caption
```

Comment:
It would be nice to show example Asciidoc markup for *include a caption* and *contain headings*.

Reply:
Good idea.  I've added a couple of examples. (I thought about linking as an alternative.)